### ISSUE TITLE
Webpack Watch in app folder

### ISSUE THREAD

**Issue by shoremark** [christianalfoni/webpack-express-boilerplate#1]
Is the boilerplate only intended to live update if the server code is updated? If I update anything in the app folder I need stop and restart server.js

**Comment by christianalfoni** [comment]
Hi @shoremark ,

sorry for the late reply.

It should update when changing files in app folder only actually. But you have to make sure those are part of the "main.js" file... so that they are part of main.js dependency tree. Could that be it?

**Comment by QuietOmen** [comment]
I believe there actually is an error somewhere in your code. I receive the following when I console out the server errors: 

```
Server Error:  { [Error: socket hang up] code: 'ECONNRESET' }
Server Error:  [Error: Must provide a proper URL as target]
```

I say this as I have spent the last couple of hours trying to track down the problem as to why my content also doesn't update and I have to stop and rerun things to get the changes to show up. I believe the answer is tied in with the following: 

https://github.com/webpack/webpack-dev-server/issues/135

Where gaearon talks about the files being served up on a different port and explicitly pointing to `http://localhost:8080/build/`. I have tried to use similar approaches with your code but I still can't seem to get it to work. I'm new to Express js and to Webpack, so it's likely that I am simply overlooking something.
